cypress/page/shopping-cart.page.ts

```diff
@@ -0,0 +1,12 @@
+class ShoppingCartPage {
+    private proceed : string;
```

Comment:
could you add to the attributes of each page object the type of element we are interacting with? for example in this case, the element is a Button